### DISCUSSION TITLE
Amazon Managed Grafana now supports Grafana version 10.4

### DISCUSSION THREAD

**Discussion by VermaPriyanka** [aws/amazon-managed-grafana-roadmap#74] [Announcements]
Customers can now run [Amazon Managed Grafana](https://aws.amazon.com/grafana/) workspaces with Grafana version 10.4. This release includes features that were launched as a part of open source Grafana versions 9.5 to 10.4, including Correlations, Subfolders, and new visualization panels such as Data Grid, XY chart and Trend panel. This release also introduces new configuration APIs to manage service accounts and tokens for Amazon Managed Grafana workspaces. [Service Accounts](https://docs.aws.amazon.com/grafana/latest/userguide/service-accounts.html), replace [API keys](https://docs.aws.amazon.com/grafana/latest/userguide/using-api-keys.html) as the primary way to authenticate applications with Grafana APIs using Service Account Tokens. These new APIs eliminate the need to manually create Service accounts, enabling customers to fully automate their provisioning workflows.

With correlations, customers can define relationships between different data sources, rendered as interactive links in Explore visualizations that trigger queries on the related data source; carrying forward data like namespace, host, or label values, enabling root cause analysis with a diverse set of data sources. Subfolders enable nested hierarchy of folders with nested layers of permissions, allowing customers to organize their dashboards to reflect their organization’s hierarchy. To explore the complete list of new features, please refer to our [user documentation](https://docs.aws.amazon.com/grafana/latest/userguide/version-differences.html#version-diff-v10).

Grafana version 10.4 is supported in all AWS regions where Amazon Managed Grafana is [generally available](https://docs.aws.amazon.com/grafana/latest/userguide/what-is-Amazon-Managed-Service-Grafana.html#AMG-supported-Regions).

You can create a new Amazon Managed Grafana workspace or upgrade your existing 9.4 workspace to 10.4 from the [AWS Console](https://console.aws.amazon.com/grafana/home/), SDK, or CLI. Check out the [Amazon Managed Grafana user guide](https://docs.aws.amazon.com/grafana/latest/userguide/version-differences.html) and [Amazon Managed Grafana API Reference](https://docs.aws.amazon.com/grafana/latest/APIReference/API_CreateWorkspaceServiceAccount.html) for detailed documentation.

**Want to learn more?**
• [What’s New](https://aws.amazon.com/about-aws/whats-new/2024/05/amazon-managed-grafana-supports-version-10-4/)
• [User Guide](https://docs.aws.amazon.com/grafana/latest/userguide/version-differences.html#version-diff-v10)
• [Blog Post](https://aws.amazon.com/blogs/mt/amazon-managed-grafana-announces-support-for-grafana-version-10-4/)

**Feedback**
To share any feedback about this feature, please leave a comment in this discussion or submit a new issue.

**Comment by JohnLBevan** [comment]
@VermaPriyanka Do you know if there are policies around how long after a new version of Grafana becomes availablel it would be supported by AWS?  i.e. for 10.4 it looks like you had it supported within 3 months of release (good work!); is that an SLA, or is there no policy; but rather goverened by complexity vs demand?  Thanks in advace, and apologies is this is not the place for such questions.

**Comment by VermaPriyanka** [comment]
@JohnLBevan MG is working towards establishing a cadence for new versions support and doesn't have a set SLA at the moment. It may not support all minor versions, but plans to establish a predictable cadence of major and minor version releases.

**Reply by CraigLager** [reply]
any update on this?

**Reply by bafulton** [reply]
Any update on this?

**Reply by adiospeds** [reply]
Hi @VermaPriyanka  .. any update on this ? Looking forward to Grafana 11 as it supports better features for Cloudwatch queries.

**Reply by JohnLBevan** [reply]
RE v11 it sounds like it's due before the end of this calendar year: https://github.com/aws/amazon-managed-grafana-roadmap/issues/94#issuecomment-3032821217
v12 shouldn't be far behind, but no explicit timeline given.

This thread mentions v11.6, which is the latest minor version for v11: https://github.com/aws/amazon-managed-grafana-roadmap/issues/21
